components/body/index.styl

```diff
@@ -1,89 +1,200 @@
-.body-content-container
-  padding-top 20px
-  garamond-size('l-body')
-
-.main-side-margin
-  margin 0 20px
-
-.body-section-title
-  margin-bottom 20px
-  avant-garde('l-headline')
-  &:after
-    content ''
-    display block
-    width 50px
-    margin 20px auto 0 auto
-    border-bottom 2px solid black
-
-
-.margin-bottom
-  margin-bottom 40px
-
-.body-section-content img
-  width 100%
-  margin-bottom 15px
-  padding-top 40px
+@media (max-width 480px)
+  .body-content-container
+    padding-top 20px
+    garamond-size('l-body')
+
+  .main-side-margin
+    margin 0 20px
+
+  .chevron-nav-list section > a
```

Comment:
Not a big deal, but similar to [this comment](https://github.com/artsy/continuum/pull/4#discussion_r37795645). Since this is selecting the bare `.chevron-nav-list` class (which is a component of Elan) it should ideally live outside of other specific components like `body`. Ideally _all_ selectors inside of a component's stylesheets are namespaced by the component name or scoped by a namespaced selector e.g. `.body-content-container .chevron-nav-list` is legit too. The exception that can be made here is the "global" component like `layout`.

Again, not going to be an issue for a small project like this, but for bigger projects like Force, being rigorous about this is what avoids CSS spaghetti by enforcing no conflicts.